src/components/accessible-autocomplete/accessible-autocomplete.yaml

```diff
@@ -0,0 +1,116 @@
+params:
+  - name: select
+    type: string
+    required: true
+    description: The select element.
+  - name: id
+    type: string
+    required: true
+    description: The id and for of the input.
+  - name: errorMessage
+    type: string
+    required: false
+    description: Options for the error message component. The error message component will not display if you use a falsy value for `errorMessage`, for example `false` or `null`.
+  - name: hint
+    type: object
+    required: false
+    description: Options for the hint component.
+  - name: describedBy
+    type: string
+    required: false
+    description: One or more element IDs to add to the `aria-describedby` attribute, used to provide additional descriptive information for screenreader users.
+
+examples:
```

Comment:
@kyle-bowden just had a thought, could we have an example with just the data-module attribute, so none of the other data attribute options provided? We have examples of different combinations of attributes but not one where the optional ones are omitted all together

Reply:
I have added an additional example that takes a select with no optional data attributes, just the `data-module="hmrc-accessible-autocomplete"` attribute. For the integration test, I test that the enhanceSelectElement generates an input with all default values, as I believe that should be the behavior.